superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -852,25 +852,36 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
     setTabKey(key);
   };
 
-  const renderStepTwoAlert = () =>
-    db?.engine && (
-      <StyledAlertMargin>
-        <Alert
-          closable={false}
-          css={(theme: SupersetTheme) => antDAlertStyles(theme)}
-          type="info"
-          showIcon
-          message={
-            engineSpecificAlertMapping[db.engine]?.message ||
-            connectionAlert?.DEFAULT?.message
-          }
-          description={
-            engineSpecificAlertMapping[db.engine]?.description ||
-            connectionAlert?.DEFAULT?.description
-          }
-        />
-      </StyledAlertMargin>
+  const renderStepTwoAlert = () => {
+    const path = window.location.hostname;
+    let ipAlert = connectionAlert.REGIONAL_IPS.default;
```

Comment:
We should add some guards here, no? Something like:

```suggestion
    let ipAlert = connectionAlert?.REGIONAL_IPS?.default || '';
```